packages/react-core/src/components/InputGroup/InputGroupText.tsx

```diff
@@ -2,35 +2,29 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/InputGroup/input-group';
 import { css } from '@patternfly/react-styles';
 
-export enum InputGroupTextVariant {
-  default = 'default',
-  plain = 'plain'
-}
-
 export interface InputGroupTextProps extends React.HTMLProps<HTMLSpanElement | HTMLLabelElement> {
   /** Additional classes added to the input group text. */
   className?: string;
   /** Content rendered inside the input group text. */
   children: React.ReactNode;
   /** Component that wraps the input group text. */
   component?: React.ReactNode;
-  /** Input group text variant */
-  variant?: InputGroupTextVariant | 'default' | 'plain';
 }
 
 export const InputGroupText: React.FunctionComponent<InputGroupTextProps> = ({
   className = '',
   component = 'span',
   children,
-  variant = InputGroupTextVariant.default,
   ...props
 }: InputGroupTextProps) => {
   const Component = component as any;
   return (
     <Component
       className={css(
+        styles.inputGroupItem,
         styles.inputGroupText,
-        variant === InputGroupTextVariant.plain && styles.modifiers.plain,
+        styles.modifiers.plain,
+        styles.modifiers.box,
```

Comment:
This is causing differences in the markup compared to Core. In Core the markup is:

![Core markup for InputGroup](https://user-images.githubusercontent.com/70952936/232513302-3b6bcc52-335d-46e3-8e87-e0958539f5fd.png)


This PR the markup becomes:

![PR preview markup for InputGroup](https://user-images.githubusercontent.com/70952936/232512981-eee553a0-acbf-463b-904b-8a0c884fd0ae.png)